tensorflow/python/ops/image_ops_impl.py

```diff
@@ -640,6 +640,18 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example:
+  
+  >>> image = tf.random.normal(shape=(3, 2, 3))  
```

Comment:
Please use `tf.constant`.

Random numbers don't help demonstrate what transpose really does. Also, doctests will fail on every run since random values will change.

So, I would suggest passing in an array and then using transpose on it and then showing the output.

Similarly comment for everywhere else.

Reply:
I have used `tf.constant` now they are no longer random numbers from the output of the transpose and others you will see that the shapes are different implying that it has been transposed and the output does not changes